src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Intake extends SubsystemBase {
+    private static Intake INSTANCE = null;
+    private IntakeIO io = null;
```

Comment:
This should be final, and there's no reason to make it null.

Reply:
07e25748beec872ac504efaf734403e83b8534cd

Reply:
a2668c70dab52d8f6b7caac96cebb4f97cf1a4ae